lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/service/LessonBuilderAccessService.java

```diff
@@ -697,7 +697,17 @@ public void handleAccess(HttpServletRequest req, HttpServletResponse res, Refere
 							rfc1123Date.setTimeZone(TimeZone.getTimeZone("GMT"));
 							res.addHeader("Last-Modified", rfc1123Date.format(lastModTime));
 						}
-						
+
+						// There is a problem embedding wav files and using the "audio/vnd.wave" mime type
+						// as the audio is rendered twice, on an <audio> tag and on an <object> tag for preventing
+						// the audio player tag could not work for that type of file, "audio/vnd.wave" mime type
+						// is being embedding before checking if it is a playable file for the audio player.
+						// changing the mime type to "audio/x-wav" will prevent the download of the file.
+						// "audio/vnd.wave", "audio/wav" and "audio/x-wav" should aslo get included into "content.mime.inline" property
+						if (contentType.equalsIgnoreCase("audio/vnd.wave")) {
```

Comment:
Invert the values in the comparison, please, or use StringUtils.equalsIgnoreCase. Thanks!

Reply:
Done! Thanks @mpellicer